app/views/pages/moving-guide/claims.html.erb

```diff
@@ -0,0 +1,198 @@
+    </div>
+
+    <div class="what-to-expect-section" id="lost-damaged-items">
+      <h4>Lost &amp; Damaged Household Goods Items</h4>
+
+      <p>If any of your household goods are lost or damaged during the moving process, you are entitled you to the full replacement value of that item, assuming your claim is filed in a timely manner. <strong>All claims must be submitted on the <%= link_to 'Defense Personal Property System (DPS)', 'https://eta.sddc.army.mil/ETASSOPortal/default.aspx' %>, which is the portal your move originally was scheduled in.</strong> However, if you and your transportation service provider (TSP) cannot negotiate a settlement on those items, you should later transfer the claim to your your <%= link_to 'Military Claims Office', page_path( 'customer-service', anchor: 'claims') %> so that they can help.</p>
+
+      <div class="usa-media_block pro-tip">
+        <%= image_tag 'icons/pro-tip.svg', alt: 'lightbulb icon', class: 'usa-media_block-img' %>
+        <div class="usa-media_block-body">
+          <strong>Pro-Tip:</strong> You can only accept compensation for a single item once during the moving process, so be careful when you negotiate or accept payment with your transportation service provider. Sometimes, if something is immediately recognized as damaged on delivery, the moving company will offer you up to $500 that day, which is called a “quick claim”. If you accept that payment, you cannot file claims on those items later on to recieve the full replacement value.
+        </div>
+      </div>
+    </div>
+
+    <div class="what-to-expect-section" id="full-replacement-value">
+      <h4>What does "Full Replacement" Value" mean?</h4>
+
+      <p>When you are moving with the military, your items are insured at “Full Replacement Value” at no additional cost to the service family. Essentially, this means that if an item is lost or destroyed during the move, the carrier should pay to replace it with a like item. The replacement could be new, used, or if the item is damaged repaired to fulfill the “Full Replacement Value” policy. The carrier will still require proof of the item's value and quality and will likely want evidence of the cost to replace it.</p>
+    </div>
+
+    <div class="what-to-expect-section" id="report">
+      <h4>Loss & Damage Report</h4>
+
+      <p>A Loss/Damage Report allows you to notify the moving company of your intent to file a claim for missing or damaged items. The big advantage of filing the Loss/Damage report is that it extends the amount of time you can file claims and still receive the full replacement value. <strong>Remember… a Loss/Damage Report is NOT a claim, you must still file a itemized claim to receive reimbursement!</strong></p>
```

Comment:
The last phrase of "Remember… a Loss/Damage Report is NOT a claim, you must still file a itemized claim to receive reimbursement!" should start with either a semi-colon or "and" to avoid being a run-on sentence. E.g., "Remember… a Loss/Damage Report is NOT a claim; you must still file a itemized claim to receive reimbursement!"